packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -173,11 +173,11 @@ class Send extends Component {
       return { amount: 'Please enter a non-zero amount' };
     } else if (amountBn.isNegative()) {
       return { amount: 'Please enter a positive amount' };
-    } else if (token.symbol === 'ETH' && toWei(values.amount).lt(1)) {
+    } else if (token.address === 'ETH' && toWei(values.amount).lt(1)) {
```

Comment:
`token.address`(i.e. `0x005072Fb985cc64d8A1Ce00ECe26279f73c12F8a`) should be `token.symbol` (i.e. `ETH`)